### DISCUSSION TITLE
Release 1.1.1 2024-06-23

### DISCUSSION THREAD

**Discussion by lcn2** [ioccc-src/mkiocccentry#872] [Announcements]
Changed `LOCATION_VERSION` to `"1.0.4 2024-05-22"`.

Fixed usage string in `location_main.c`: it did not suggest that one can specify
more than one location but it can.

Added to the README.md brief documentation of the `location(1)` tool (it was not
there at all) and fixed some typos and added some clarity in the `location.1`
man page.

Updated the Rule 2a size from 4096 to 4993: a change that keeps the
"Rule 2b:Rule 2a" ratio (that was recently changed from 2053 to 2503)
similar to the 2001-2020 IOCCC era.

<hr /><em>This discussion was created from the release <a href='https://github.com/ioccc-src/mkiocccentry/releases/tag/v1.1.1'>Release 1.1.1 2024-06-23</a>.</em>

**Comment by SirWumpus** [comment]
> Updated the Rule 2a size from 4096 to 4993: a change that keeps the
"Rule 2b:Rule 2a" ratio (that was recently changed from 2053 to 2503)
similar to the 2001-2020 IOCCC era.

Has it really been 19 years for this era?!  Simply WOW!  How times change.  (Insert four Yorkshiremen sketch.)
https://www.youtube.com/watch?v=ue7wM0QC5LE  I remember when I had to consider writing an entry in -1536 bytes, BEFORE imaginary numbers!

**Reply by xexyl** [reply]
> > Updated the Rule 2a size from 4096 to 4993: a change that keeps the
> > "Rule 2b:Rule 2a" ratio (that was recently changed from 2053 to 2503)
> > similar to the 2001-2020 IOCCC era.
> 
> Has it really been 19 years for this era?! Simply WOW! How times change. (Insert four Yorkshiremen sketch.) https://www.youtube.com/watch?v=ue7wM0QC5LE I remember when I had to consider writing an entry in -1536 bytes, BEFORE imaginary numbers!

You started before me by many years of course, though I was into programming at the time, but the first time for me was as you probably know 2053. But on the subject of imaginary numbers here's a great cartoon for you:

![IMG_2421](https://github.com/ioccc-src/mkiocccentry/assets/53008573/c569d84c-d1e6-48b6-a160-674e2759d33b)

**Comment by xexyl** [comment]
Thanks, Landon, for updating the version after that change I made. I did not think to do that. Good call.

**Comment by lcn2** [comment]
Mind this: The current [rules](https://ioccc-src.github.io/temp-test-ioccc/next/rules.html) and [guidelines](https://ioccc-src.github.io/temp-test-ioccc/next/guidelines.html) are still tentative

Nevertheless we try to keep this repo in sync with tentativeness 😉

**Reply by xexyl** [reply]
> Mind this: The current [rules](https://ioccc-src.github.io/temp-test-ioccc/next/rules.html) and [guidelines](https://ioccc-src.github.io/temp-test-ioccc/next/guidelines.html) are still tentative
> 
> Nevertheless we try to keep this repo in sync with tentativeness 😉

Ever notice that the words TENTATIVE and ATTENTIVE are anagrams? I just noticed it as I looked at the comment .. as I have done with many other anagrams I have noticed. Some amusing ones I have of course shared with you before.

**Reply by SirWumpus** [reply]
Along with `Eaten It TV` that brings you shows like `Food for thought`, `Gluttony with kale`, and `Burps: Cow vs Man III`.

**Reply by xexyl** [reply]
> Along with `Eaten It TV` that brings you shows like `Food for thought`, `Gluttony with kale`, and `Burps: Cow vs Man III`.

One I have shared with Landon that's brilliant: LISTEN = SILENT. If you want to be a good listener you should keep your mouth shut!

Some that are not my own discovery but are great and one being the best I know of (though I have noticed some that are quite similar .. plus words within words, words spelt backwards and other things):

Dormitory = Dirty room (!!)

The Morse code = Here come dots (also excellent irony!)

Slot machines = Cash lost in 'em (not as big on this one as it's not a full word but it's still kind of fun .. though I'd be more likely to call them a fruit machine)

Snooze alarms = Alas! No more z's (again not as big on this one)

Eleven plus two = Twelve plus one (my favourite because mathematically they are equal but they are also anagrams!)

**Reply by xexyl** [reply]
The funniest licence / number plate I have seen is `OCD-OCD`. I also saw one years before that I loved: `UNX-RLZ`. Actually I was with my mum at the time and we both saw it and both really enjoyed it!  I have also seen `RSYNC` and I have  heard of one like `LNX-GRL` or something like that. Still the `OCD-OCD` one is so great ...

**Reply by xexyl** [reply]
> > Along with `Eaten It TV` that brings you shows like `Food for thought`, `Gluttony with kale`, and `Burps: Cow vs Man III`.
> 
> One I have shared with Landon that's brilliant: LISTEN = SILENT. If you want to be a good listener you should keep your mouth shut!
> 
> Some that are not my own discovery but are great and one being the best I know of (though I have noticed some that are quite similar .. plus words within words, words spelt backwards and other things):
> 
> Dormitory = Dirty room (!!)
> 
> The Morse code = Here come dots (also excellent irony!)
> 
> Slot machines = Cash lost in 'em (not as big on this one as it's not a full word but it's still kind of fun .. though I'd be more likely to call them a fruit machine)
> 
> Snooze alarms = Alas! No more z's (again not as big on this one)
> 
> Eleven plus two = Twelve plus one (my favourite because mathematically they are equal but they are also anagrams!)

BTW on the subject of dormitories the etymology of CEMETERY is interesting: `late Middle English: via late Latin from Greek koimētērion ‘dormitory’, from koiman ‘put to sleep.`
